src/utils/resources/bootableresources/hooks/useHideDeprecatedBootableVolumes.ts

```diff
@@ -17,22 +13,15 @@ const useHideDeprecatedBootableVolumes: UseHideDeprecatedBootableVolumes = (
   onFilterChange,
   currentTab,
 ) => {
-  const { setParam } = useURLParams();
-
   useEffect(() => {
     if (currentTab !== CREATE_VM_TAB.INSTANCE_TYPES && currentTab !== undefined) return null;
 
-    // This is to select the 'Hide deprecated bootable volumes' filter by default
-    onFilterChange(HIDE_DEPRECATED_BOOTABLE_VOLUMES, {
-      all: [HIDE_DEPRECATED_BOOTABLE_VOLUMES],
-      selected: [HIDE_DEPRECATED_BOOTABLE_VOLUMES],
+    // Calculate the filtered bootable volumes once causing deprecated volumes to be hidden by default
```

Comment:
I think we don't need this hook anymore right ?

Reply:
In my testing the deprecated volumes show on load without the hook. If you select and then deselect the "Show deprecated bootable volumes" filter it hides them as expected. The hook triggers the filtering mechanism, which allows them to be hidden by default.